### ISSUE TITLE
phase 2 Neural networks

### ISSUE THREAD

**Issue by MahdiEsrafili** [ZahraDehghani99/AI-Internship#1]
- [x] more data augmentation and transforms
- [x] make intel classification network bigger
- [x] plot loss after training
- [x] use tensorboard for monitoring
- [x] use learning rate scheduler
- [x] use batch normalization and drop out layer in network
- [x] sample data for training
- [x] user pytorch `DatasetFolder` to create Intel image dataset. [link](https://pytorch.org/vision/0.8/datasets.html#datasetfolder)
- [x] user `set_description`  for `tqdm` to print loss and other info
